### PR TITLE
feat(codemod): UseSession Hook Update ( nextAuth - migration to v4) 

### PR DESCRIPTION
**Pull request by manishjha-04**

### CODE REVIEW

File: packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://codemod-utils.s3.us-west-1.amazonaws.com/configuration_schema.json",
+  "name": "nextAuth/v4/useSession-array-to-object-destructuring",
```
Comment (DmytroHryshyn) [inline packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json:3]: Lets change the name to `nextAuth/4/useSession-array-to-object-destructuring`

File: packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "1.0.0",
+  "engine": "jscodeshift",
+  "private": false,
+  "arguments": [],
+  "meta": {
+    "tags": ["nextAuth"],
+    "git": "https://github.com/codemod-com/codemod"
```
Comment (DmytroHryshyn) [inline packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json:10]: Lets update the `git` field to `https://github.com/codemod-com/codemod/tree/main/packages/codemods/nextAuth/useSession-array-to-Object-destrucutring`

**Comment by DmytroHryshyn** [comment]
Hi @manishjha-04 Thanks for your PR! Could you please rebase your branch on the latest `main` and then regenerate `pnpm-lock` file by running `pnpm install` once more locally?

File: packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json (hunk 1 above)
Comment (manishjha-04) [reply packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json]: Sure✌️

**Comment by manishjha-04** [comment]
@DmytroHryshyn Done👍

**Review by DmytroHryshyn — APPROVED** [review]

**Comment by DmytroHryshyn** [comment]
@manishjha-04 Thanks! Last 2 things: linter fails because of incorrect codemod files formatting (can be fixed by `pnpm lint:write`) and unit test did not pass because jscodeshift slightly changed formatting. When this is fixes, we can merge the codemod

**Pull request merged**